Terminal.Gui/Drawing/SolidFill.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Terminal.Gui.Drawing;
```

Comment:
For v2 I do want to revisit our namespaces. However, we're not there yet, so can you please make this "Terminal.Gui"?

Reply:
Fixed in 1ba84ded9a800206a1724a92980d634522c19bc7 was accident IDae copy paste